api/v1alpha1/tracing_types.go

```diff
@@ -114,3 +116,20 @@ type RequestHeaderCustomTag struct {
 	// +optional
 	DefaultValue *string `json:"defaultValue,omitempty"`
 }
+
+// ZipkinTracingProvider defines the Zipkin tracing provider configuration.
+type ZipkinTracingProvider struct {
+	// CollectorHostname defines the hostname to use when sending spans
+	// to the Zipkin collector endpoint.
+	// +optional
+	CollectorHostname *string `json:"collectorHostname,omitempty"`
+	// TraceID128Bit determines whether a 128bit trace id will be used
+	// when creating a new trace instance. If set to false, a 64bit trace
+	// id will be used.
+	// +optional
+	TraceID128Bit bool `json:"traceId128Bit,omitempty"`
```

Comment:
```suggestion
	TraceID128Bit *bool `json:"traceId128Bit,omitempty"`
```

let's follow the API guide, if it's optional, use a point.

Reply:
Updated